msteams-platform/concepts/build-and-test/apps-package.md

```diff
@@ -54,14 +54,18 @@ Teams automatically crops your icon to display a square with rounded corners in
 
 ### Outline icon
 
-An outline icon displays in two scenarios:
+An outline icon displays when your app is in use and/or pinned to the app bar on the left side of Teams.
 
-* When your app is in use and “hoisted” on the app bar on the left side of Teams.
-* When a user pins your app's message extension.
+|Platform  |Scenario  | Required|
```

Comment:
Instead of "Platform", let's call this "Microsoft 365 host application" to maintain consistent wording with other cross-M365 content